runtime/ts/storage/in-memory-storage.ts

```diff
@@ -155,22 +148,39 @@ abstract class InMemoryStorageProvider extends StorageProviderBase {
     }
     return new InMemoryVariable(type, storageEngine, name, id, key);
   }
+
+  async ensureBackingStore() {
+    if (this.backingStore) {
+      return this.backingStore;
```

Comment:
Missing await?  (Or, since we know backingStore is a non-Promise value, does the other instance have an unnecessary await?)

Reply:
Figured out how to get rid of the extra await :)